z390test/src/test/groovy/org/z390/test/RunAsmDemos.groovy

```diff
@@ -0,0 +1,42 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+class RunAsmDemos extends z390Test {
+
+    var options = ['trace', 'noloadhigh',
+                   "SYSMAC(${basePath("mac")})",
+                   "SYSOBJ(${basePath("linklib")})"]
+
+    @Test
+    void test_HELLO() {
+        int rc = this.asmlg(basePath("demo", "HELLO"), *options)
+        this.printOutput()
+        assert rc == 0
+    }
+    @Test
+    void test_DEMOM8Q1() {
+        int rc = this.asm(basePath("demo", "DEMOM8Q1"), *options)
+        this.printOutput()
+        assert rc == 0
+    }
+    @Test
+    void test_DEMODCB1() {
```

Comment:
"test_DEMODCB1()" should be "test_TESTDCB1()".

Reply:
Will fixed